src/resources/l10n/nl.json

```diff
@@ -131,7 +131,8 @@
     "attributeDeviceImage": "Apparaatafbeelding",
     "attributeDeviceInactivityStart": "Start van inactiviteit",
     "attributeDeviceInactivityPeriod": "Inactiviteitperiode",
-    "attributeDeviceDisableForwarding": "Stuur events voor device niet door",
+    "attributeDeviceAllowPositionForwarding": "Stuur posities voor device door",
+    "attributeDeviceAllowEventForwarding": "Stuur events voor device door",
```

Comment:
Do not change anything except English file. Translations should be done ONLY through Transifex.